core/xml.js

```diff
@@ -467,6 +467,7 @@ Blockly.Xml.domToWorkspace = function(xml, workspace) {
   if (workspace.setResizesEnabled) {
     workspace.setResizesEnabled(true);
   }
+  Blockly.Events.fire(new Blockly.Events.FinishedLoading(workspace));
```

Comment:
Add a `goog.require` for `Blockly.Events.FinishedLoading` at the top of this file.